sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/TraceUtil.java

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.amqp.implementation;
+
+import com.azure.core.amqp.exception.AmqpException;
+import com.azure.core.implementation.tracing.TracerProxy;
+import com.azure.core.util.Context;
+import reactor.core.publisher.Signal;
+
+/**
+ * Helper class to help start tracing spans.
+ */
+public class TraceUtil {
+
+    private static final String OPENTELEMETRY_SPAN_KEY = com.azure.core.implementation.tracing.Tracer.OPENTELEMETRY_SPAN_KEY;
```

Comment:
You could import this field statically instead of redefining another constant here.